### PR TITLE
DROOLS-4650: Reimplement equals of drools-workbench-models-guided-dtable

### PR DESCRIPTION
**Pull request by jomarko**

### CODE REVIEW

**Comment by jomarko** [comment]
@Rikkola hello, any update on this please?

File: drools-workbench-models/drools-workbench-models-guided-dtable/src/test/java/org/drools/workbench/models/guided/dtable/backend/GuidedDTDRLPersistenceTest.java [hunk 1]
```diff
@@ -1794,9 +1794,9 @@ public void testRHS() {
                      a2.getFieldValues()[0].getType());
 
         // examine the retract
-        ActionRetractFact a3 = (ActionRetractFact) rm.rhs[2];
-        assertEquals("retract",
-                     a3.getVariableName());
+//        ActionRetractFact a3 = (ActionRetractFact) rm.rhs[2];
+//        assertEquals("retract",
+//                     a3.getVariableName());
```
Comment (Rikkola) [inline drools-workbench-models/drools-workbench-models-guided-dtable/src/test/java/org/drools/workbench/models/guided/dtable/backend/GuidedDTDRLPersistenceTest.java]: This is the only part that I have questions about. Do we need to remove this or update?

**Review by Rikkola — CHANGES_REQUESTED** [review]
Thanks for reminding me. So many PR emails that I did quick check this, but didn't hit review.

File: drools-workbench-models/drools-workbench-models-guided-dtable/src/test/java/org/drools/workbench/models/guided/dtable/backend/GuidedDTDRLPersistenceTest.java (hunk 1 above)
Comment (jomarko) [reply drools-workbench-models/drools-workbench-models-guided-dtable/src/test/java/org/drools/workbench/models/guided/dtable/backend/GuidedDTDRLPersistenceTest.java]: Good question, looking.

Comment (jomarko) [reply drools-workbench-models/drools-workbench-models-guided-dtable/src/test/java/org/drools/workbench/models/guided/dtable/backend/GuidedDTDRLPersistenceTest.java]: Test is restored now.

**Review by Rikkola — APPROVED** [review]

**Pull request merged**